### PR TITLE
Adds http url validation to all weblinks

### PR DESCRIPTION
**Pull request by kevgliss**

### CODE REVIEW

**Review by whitdog47 — APPROVED** [review]

**Review by whitdog47 — CHANGES_REQUESTED** [review]
Let's see if we can get playwright fixed first

**Review by jschroth — APPROVED** [review]

**Review by whitdog47 — APPROVED** [review]

File: src/dispatch/individual/models.py [hunk 1]
```diff
@@ -63,12 +63,21 @@ class IndividualContact(Base, ContactMixin, ProjectMixin):
...
     external_id: Optional[str] = Field(None, nullable=True)
 
+    @validator("weblink")
+    def weblink_validator(cls, v):
+        if v is None:
+            return v
+        if not isinstance(v, HttpUrl) and len(v) > 0:
+            raise ValueError(
+                "weblink is not an empty string and not a valid weblink")
+        return v
```
Comment (jschroth) [inline src/dispatch/individual/models.py:79]: this is a small nit, but I prefer something that is more explicit and a raise if not.
```suggestion
        if v is None or v == "" or isinstance(v, HttpUrl):
            return v

        raise ValueError("weblink is not an empty string or a valid weblink")
```

Comment (jschroth) [reply src/dispatch/individual/models.py:79]: I forget which order these need to be in so you could reverse the last two to `isinstance(v, HttpUrl) or v == ""`

Comment (whitdog47) [reply src/dispatch/individual/models.py]: I reversed the last two. Thanks for helping clean up the logic!

**Pull request merged**